### PR TITLE
feat: Create bundle for RHEL 7.9

### PR DESCRIPTION
**Pull request by dlipovetsky**
**What problem does this PR solve?**:

**Which issue(s) does this PR fix?**:

* https://jira.d2iq.com/browse/D2IQ-NUMBER

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: bundles/redhat7.9/bundle.sh.gotmpl [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#shellcheck disable=SC2046
+repoquery  --archlist=x86_64,noarch  --resolve --requires --recursive  $(< packages.txt) | xargs -r yumdownloader --archlist=x86_64,noarch -x \*i686
+#shellcheck disable=SC2046
+yumdownloader  --resolve --archlist=x86_64,noarch -x \*i686 $(< packages.txt)
+rm packages.txt
+curl https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm -o amazon-ssm-agent.rpm
+createrepo -v .
+chown -R 1000:1000 repodata/
+#shellcheck disable=SC1083,SC2035
+tar -czf {{ .OutputDirectory }}/{{ .KubernetesVersion }}_redhat_8.4_x86_64{{ .FipsSuffix }}.tar.gz *
```
Comment (faiq) [inline bundles/redhat7.9/bundle.sh.gotmpl:63]:
```suggestion
tar -czf {{ .OutputDirectory }}/{{ .KubernetesVersion }}_redhat_7.9_x86_64{{ .FipsSuffix }}.tar.gz *
```

File: bundles/redhat7.9/bundle.sh.gotmpl [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+#shellcheck disable=SC1083
+chmod 777 {{ .OutputDirectory }}/{{ .KubernetesVersion }}_redhat_8.4_x86_64{{ .FipsSuffix }}.tar.gz
```
Comment (faiq) [inline bundles/redhat7.9/bundle.sh.gotmpl:65]:
```suggestion
chmod 777 {{ .OutputDirectory }}/{{ .KubernetesVersion }}_redhat_7.9_x86_64{{ .FipsSuffix }}.tar.gz
```

File: bundles/redhat7.9/packages.txt.gotmpl [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+python-kitchen
+python-chardet
+yum-plugin-ovl
+python2-dnf
+NetworkManager-libnm
+xmlsec1
+xmlsec1-openssl
+python3-libs
+python3-3.6.8
+perl-libs-5.16.3-297.el7
```
Comment (faiq) [inline bundles/redhat7.9/packages.txt.gotmpl:46]:
```suggestion
perl-libs-5.16.3-297.el7
elfutils-libelf-devel
```

**Review by faiq — APPROVED** [review]

File: bundles/redhat7.9/packages.txt.gotmpl (hunk 3 above)
Comment (dlipovetsky) [reply bundles/redhat7.9/packages.txt.gotmpl:46]: Thanks, fixed!

**Comment by dlipovetsky** [comment]
I squashed and rebased on the branch.

**Pull request merged**